docs/variables.md

```diff
@@ -108,14 +93,6 @@ variable via `resources.inputs.<resourceName>.<variableName>` or
 | `url` | The complete path to the image. |
 | `digest` | The digest of the image. |
 
-#### Variables for the `GCS` type
-
-| Variable | Description |
-| -------- | ----------- |
-| `name` | The name of the resource. |
-| `type` | Type value of `"gcs"`. |
-| `location` | The fully qualified address of the blob storage. |
-
 #### Variables for the `Cluster` type
```

Comment:
Can you please open a separate PR removing the docs for cluster and cloudevent resources?

Reply:
Appreciate the catch, updated at https://github.com/tektoncd/pipeline/pull/6158